v2/pgxcompat/uuid_test.go

```diff
@@ -0,0 +1,274 @@
+package pgxcompat_test
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/jackc/pgtype"
+	"github.com/jackc/pgtype/testutil"
+
+	"github.com/SKF/go-utility/v2/pgxcompat"
+	"github.com/SKF/go-utility/v2/uuid"
+)
+
+func TestUUIDTranscode(t *testing.T) {
+	testutil.TestSuccessfulTranscode(t, "uuid", []interface{}{
+		&pgxcompat.UUID{UUID: uuid.UUID("5b9cb067-8180-4953-846c-be5764532dc0"), Status: pgtype.Present},
+		&pgxcompat.UUID{Status: pgtype.Null},
+	})
+}
+
+type SomeUUIDWrapper struct {
+	SomeUUIDType
+}
+
+type SomeUUIDType [16]byte
+
+func stringPtr(str string) *string {
+	return &str
+}
+
+func TestUUIDSet(t *testing.T) {
+	successfulTests := []struct {
+		name   string
+		source interface{}
+		result pgxcompat.UUID
+	}{
+		{
+			name:   "Set to nil",
+			source: nil,
+			result: pgxcompat.UUID{Status: pgtype.Null},
+		},
+		{
```

Comment:
Can you add a test for uuid.Empty?, {
			name:   "Set empty uuid",
			&pgxcompat.UUID{UUID: uuid.Empty Status: pgtype.Present},
			result: pgxcompat.UUID{Status: pgtype.Null},
		},